### ISSUE TITLE
Last dispatch cannot be analyzed

### ISSUE THREAD

**Issue by gsitaram** [AMDResearch/omniperf#84]
There are two invocations of this kernel, omniperf collects metrics for both dispatches, but we are able to analyze only the first one. It seems to be assuming 2 header lines in pmc_perf.csv, but there is only one. If we duplicated the last dispatch and called it dispatch 2, then I can analyze dispatch 1. Please fix this so that such a workaround is not needed. 
Steps to reproduce:
- See this simple reproducer:
```
$ cat test.cpp 
#include <hip/hip_runtime.h>
__global__ void write(int N, int *in) {
    int tid = blockIdx.x*blockDim.x + threadIdx.x;
    if (tid < N) {
        in[tid] = 1;
    }
}
int main() {
  int* d_in=NULL;
  int N=1024;
  hipMalloc(&d_in, N*sizeof(int));
  write<<<(N+255)/256,256>>>(N, d_in);
  write<<<(N+255)/256,256>>>(N, d_in);
  hipDeviceSynchronize();
}
```
- Compile: `hipcc -O3 -o test test.cpp`
- Profile: `omniperf profile -n dispatch --device 0 --no-roof -- ./test`
- Analyze: `omniperf analyze -p workloads/dispatch/mi200 -b 17.2.1 --filter-dispatch-ids 0` <-- works
- Analyze: `omniperf analyze -p workloads/dispatch/mi200 -b 17.2.1 --filter-dispatch-ids 1` <-- fails

**Comment by coleramos425** [comment]
I appreciate you providing this thorough report, Gina. The bug has been fixed and will be rolled in v1.0.7

I'll notify you when it becomes available. Shouldn't be more than a few days.

**Comment by gsitaram** [comment]
Thank you, Cole, for fixing this.
